Dockerfile.dev.win

```diff
@@ -0,0 +1,25 @@
+# escape=`
+
+ARG WINDOWS_VERSION
+FROM docker-registry.devs.facilities.rl.ac.uk/bisapps-node:12-servercore${WINDOWS_VERSION}
+
+#Need permissions to add nessary fonts so chromium can launch
+USER ContainerAdministrator
+
+#Get imagemagick, get all link in the binarys folder then regex to find the latest release
```

Comment:
May be worth adding a note here (and in the other Dockerfile) that ImageMagick only hosts Windows builds for the current version. That would explain why we can't download a fixed version, which is generally how Dockerfiles work.

Reply:
Sounds good I'll expand on this comment